tests/functional/commands/validate/lib/models/function_with_event_filtering.yaml

```diff
@@ -4,7 +4,7 @@ Resources:
     Properties:
       CodeUri: s3://sam-demo-bucket/filtered_events.zip
```

Comment:
If `filtered_events.zip` uses Lambda Provided JS SDK v2, this test will fail.

The JS SDK team recommends bumping to v3, and deploying the dependency in node_modules instead of depending on Lambda Provided one - which is provided for convenience.